app/models/dashboard.rb

```diff
@@ -40,7 +41,20 @@ def closed_defence_requests
   attr_reader :user, :defence_requests_scoped_by_policy
 
   def ordered_defence_requests
-    defence_requests_scoped_by_policy.ordered_by_created_at
+    defence_requests_scoped_by_policy.ordered_by_created_at.map { |d| new_dash_dr[d] }
+  end
+
+  def new_dash_dr
+    @_new_dash_dr ||= DefenceRequest.method(:new).curry(2)[@client]
```

Comment:
I believe I am at fault for this. I usually prefix ivars with an underscore to reveal the intent they are not meant to be used in views. I've found it a helpful distinction in controllers and just use elsewhere to be consistent. If the team prefers to lose the prefix, I'm happy with that :)